temp.py

```diff
@@ -0,0 +1,2 @@
+import pdb
+print("hello")
```

Comment:
Avoid using `print` statements for logging in production code. Consider using a logging framework to handle log messages.
```
Replace `print("hello")` with a logging statement using Python's logging module.
```python
import logging
logging.basicConfig(level=logging.INFO)
logging.info("hello")
```
```